modules/custom/d_social_media/templates/social_media.html.twig

```diff
@@ -0,0 +1,18 @@
+{% if show %}
+  {{ attach_library('d_social_media/last-element-in-a-row') }}
+
+  <div class='social-media-wrapper'>
+    <div class='follow-us'>Follow us</div>
+    <div class='icons-wrapper'>
+      <ul>
+      {% for link in links %}
+        <li><a href='{{ link.link }}' class='icon icon-{{ link.name }}' target="_blank"></a></li>
```

Comment:
What if one of the link is empty? It should not be visible

Reply:
It is never empty. Array $links is created in class SocialMediaBlock from non-empty configuration elements.